galeracluster/source/kb/gaps-in-gtid-sequence-numbers.rst

```diff
@@ -0,0 +1,145 @@
+.. meta::
+   :title: Gaps in GTID Sequence Numbers
+   :description:
+   :language: en-US
+   :keywords:
+   :copyright: Codership Oy, 2014 - 2024. All Rights Reserved.
+
+
+.. container:: left-margin
+
+   .. container:: left-margin-top
+
+      :doc:`The Library <../index>`
+
+   .. container:: left-margin-content
+
+      - :doc:`Documentation <../documentation/index>`
+
+      .. cssclass:: here
+
+         - :doc:`Knowledge Base <./index>`
+
+      - :doc:`Training <../training/index>`
+
+      .. cssclass:: sub-links
+
+         - :doc:`Training Courses <../training/courses/index>`
+         - :doc:`Tutorial Articles <../training/tutorials/index>`
+         - :doc:`Training Videos <../training/videos/index>`
+
+      - :doc:`FAQ <../faq>`
+      - :ref:`search`
+
+      Related Documents
+
+      - :doc:`Schema Upgrades <../../documentation/schema-upgrades>`
+      - :ref:`wsrep_OSU_method <wsrep_OSU_method>`
+
+.. container:: top-links
+
+   - `Home <https://galeracluster.com>`_
+   - :doc:`Docs <../documentation/index>`
+
+   .. cssclass:: here
+
+      - :doc:`KB <./index>`
+
+   .. cssclass:: nav-wider
+
+      - :doc:`Training <../training/index>`
+
+   - :doc:`FAQ <../faq>`
+
+
+.. cssclass:: library-article
+.. _`kb-trouble-gaps-in-gtid-sequence-numbers`:
+
+================================
+Gaps in GTID Sequence Numbers
+================================
+
+.. rst-class:: article-stats
+
+   Length: 421 words; Published: October 22, 2024; Updated: October 22, 2024; Category: State Transfers; Type: Troubleshooting
+
+GTID gaps may occur if ``gtid_next`` is set to a specific value, and a transaction fails.
+
+.. rst-class:: section-heading
+.. rubric:: Scenario
+
+When a failing DDL statement with a GTID seqno is replicated by TOI, the failed DDL is not logged in the binary log and, therefore, the binlog will contain GTID event gaps.
```

Comment:
You'd better start like that:
"When a failing DDL statement with explicit GTID value set with `gtid_next` is replicated ..."

Note: technically, any DDL or DML is affected with explicit `gtid_next` setting, but as an example scenario the current version looks OK.